include/CMakeLists.txt

```diff
@@ -12,8 +12,8 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-target_include_directories(VulkanLayerUtils PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>)
+target_include_directories(VulkanLayerSettings PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>)
 
-target_compile_features(VulkanLayerUtils PUBLIC cxx_std_17)
+target_compile_features(VulkanLayerSettings PUBLIC cxx_std_17)
```

Comment:
Looking at vk_layer_settings.h this can be removed now. Since vk_layer_setting.h exposes a C interface now.

Instead it should now be `target_compile_features(VulkanLayerSettings PRIVATE cxx_std_17)` and moved to the `src` directory since it's an implementation detail.

Reply:
This is resolved with last commit